Sources/PackageCollectionsSigning/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+add_library(PackageCollectionsSigning
+  Certificate/Certificate.swift
```

Comment:
Looks like the cmake build works fine for https://github.com/apple/swift-package-manager/pull/3202 because I forgot to add the source file to the list. With this PR however, I'm seeing this error locally:

```
--- bootstrap: note: Building SwiftPM (with CMake)
[1/1] Linking Swift shared library lib/libPackageCollectionsSigning.so
FAILED: lib/libPackageCollectionsSigning.so Sources/PackageCollectionsSigning/CMakeFiles/PackageCollectionsSigning.dir/Certificate/Certificate.swift.o Sources/PackageCollectionsSigning/CMakeFiles/PackageCollectionsSigning.dir/Key/BoringSSLKey.swift.o Sources/PackageCollectionsSigning/CMakeFiles/PackageCollectionsSigning.dir/Key/Key.swift.o Sources/PackageCollectionsSigning/CMakeFiles/PackageCollectionsSigning.dir/Key/Key+EC.swift.o Sources/PackageCollectionsSigning/CMakeFiles/PackageCollectionsSigning.dir/Key/Key+RSA.swift.o Sources/PackageCollectionsSigning/CMakeFiles/PackageCollectionsSigning.dir/Utilities/Utilities.swift.o swift/PackageCollectionsSigning.swiftmodule 
: && /usr/bin/swiftc -output-file-map Sources/PackageCollectionsSigning/CMakeFiles/PackageCollectionsSigning.dir/output-file-map.json -incremental -j 6 -emit-library -o lib/libPackageCollectionsSigning.so -module-name PackageCollectionsSigning -module-link-name PackageCollectionsSigning -emit-module -emit-module-path swift/PackageCollectionsSigning.swiftmodule -emit-dependencies -DPackageCollectionsSigning_EXPORTS -module-cache-path "/code/swift-package-manager/.build/x86_64-unknown-linux-gnu/bootstrap/module-cache" -g -swift-version 5 -I ../swift-crypto/swift ../../../Sources/PackageCollectionsSigning/Certificate/Certificate.swift ../../../Sources/PackageCollectionsSigning/Key/BoringSSLKey.swift ../../../Sources/PackageCollectionsSigning/Key/Key.swift ../../../Sources/PackageCollectionsSigning/Key/Key+EC.swift ../../../Sources/PackageCollectionsSigning/Key/Key+RSA.swift ../../../Sources/PackageCollectionsSigning/Utilities/Utilities.swift  -Xlinker -soname -Xlinker libPackageCollectionsSigning.so  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/swift-crypto/lib  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/swift-crypto/lib  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/swift-crypto/lib  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/swift-crypto/lib   -L /usr/lib/gcc/x86_64-linux-gnu/9 -Xlinker -rpath -Xlinker /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/swift-crypto/lib  ../swift-crypto/lib/libCrypto.so  ../swift-crypto/lib/libCCryptoBoringSSL.a  -ldispatch  -lFoundation  ../swift-crypto/lib/libCCryptoBoringSSLShims.a  ../swift-crypto/lib/libCCryptoBoringSSL.a  -lgcc  -lgcc_s  -lc  -lgcc  -lgcc_s && :
../../../Sources/PackageCollectionsSigning/Certificate/Certificate.swift:15:29: error: no such module 'CCryptoBoringSSL'
@_implementationOnly import CCryptoBoringSSL
```

It seems to be a matter of whether or not the source files that import `CCryptoBoringSSL` are on this list. If I comment those out the build completes successfully.

@compnerd How can we fix this?

Reply:
https://github.com/apple/swift-crypto/pull/71 should help; since `CCryptoBoringSSL` is a user visible module, we just need to distribute the headers and libraries for use here.

Reply:
Confirmed locally that https://github.com/apple/swift-crypto/pull/71 fixes the build. Thanks!